lib/klaviyo/client.rb

```diff
@@ -49,6 +49,15 @@ def self.v1_request(method, path, kwargs = {})
       request(method, full_url)
     end
 
+    # V1 Post requests use x-www-form-urlencoded content instead of JSON
+    def self.v1_post_request(path, params = {})
+      query_params = encode_params(params)
+      path = "/api/#{V1_API}/#{path}"
```

Comment:
use v1_request, add a new kwarg for content type that can get passed down into the request method